blueprints/app/files/app/index.html

```diff
@@ -15,6 +15,10 @@
     {{content-for "head-footer"}}
   </head>
   <body>
+    <noscript>
+      JavaScript must be enabled to run this application.
```

Comment:
I am not sure I understand what extra work FastBoot takes? It is supposed to help in your initial page renders rather than the other way round. If people would remove it, I would like to first understand why they would do so?